### DISCUSSION TITLE
How does the coloring details field in custom overlays function?

### DISCUSSION THREAD

**Discussion by ChildishGiant** [Helium314/SCEE#501] [Q&A]
I can't find any hints or documentation on what to put there to make certain elements certain colours.

**Comment by Helium314** [comment]
You can click the "Coloring detail" text for some infos.

**Reply by mnalis** [reply]
I've missed that possibility too! Probably because the orange coloring of that `Coloring detail` (at least in dark theme) looks just like regular "section name" (e.g. `Communications`, `Display`, `SCEE settings`, ... ) in the menus, which are _not_ clickable.  

Maybe we could add "&#9432;"  at the end of texts describing clickable links?  Or some other clear UI indication what elements are clickable?

**Reply by Helium314** [reply]
This was already added several weeks ago, so will be in the next release
